docs/src/modeler/example_dynamic_data.md

```diff
@@ -0,0 +1,335 @@
+# Creating a Dynamic System
+
+This tutorial briefly introduces how to create a system using `PowerSystems.jl` data structures. The tutorial will guide you to create the JSON data file for the tutorial 1. We start by calling `PowerSystems.jl`:
```

Comment:
This should said tutorial 1 in PowerSimulationDynamics.jl

Reply:
This is a bit of a place holder for now. We probably want to rephrase this better